src/Mvc/HttpCodeConvention.cs

```diff
@@ -50,6 +50,16 @@ public static class HttpCodeConvention
 	/// </summary>
 	public static string ConflictProblemDetailsTitle { get; set; } = "Conflict with the current state";
 
+	/// <summary>
+	///		The HTTP code to return when failed due to a conflict with the current state of the target resource (<see cref="DomainOperationStatus.Conflict"/> status). The default value: 409
```

Comment:
Incorrect comment.
Needs to be `The HTTP code to return when the requested entity is larger than limits defined by server (<see cref="DomainOperationStatus.ContentTooLarge"/> status). The default value: 413`